src/core/util/StringBuilder.ts

```diff
@@ -30,4 +30,8 @@ export default class StringBuilder {
     public toString(): string {
         return this.value;
     }
+
+    public insert(n: number, c: string) {
```

Comment:
Just to know, was this based on `Java.lang.StringBuilder.insert()` Method? (just wondering if this could be confusing to Java devs)

Reply:
Yes, exactly this method was used in java code. It is supposed to be a port of this method.